StylableSwiftUI/Classes/UIKit/Stylist+UIKit.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  Stylist+UIKit.swift
+//  StylableSwiftUI
+//
+//  Created by Kerr Marin Miller on 01/09/2020.
+//
+
+import Foundation
+
+public enum StylistProperty {
+    case backgroundColor(UIColor)
+    case textColor(UIColor)
+    case kerning(Double)
+    case font(UIFont)
+}
+
+extension Array where Iterator.Element == StylistProperty {
+    func firstBackgroundColor() -> UIColor? {
+        return self.compactMap { prop in
+            switch prop {
+            case .backgroundColor(let color):
+                return color
+            default:
+                return nil
+            }
+        }.first
```

Comment:
Almost certainly not an issue unless we have a lot of properties, but this will run through all the properties and map each one, before getting the first.

`self.lazy.compactMap { ... }.first` would only compact map until a match was found - but I don't know the cost of making a sequence lazy vs. how many properties you're planning to use :)

Reply:
I'll make a new PR with this change